### PR TITLE
 Patch version of #48388

### PR DESCRIPTION
**Pull request by alan-agius4**
Patch version of https://github.com/angular/angular/pull/48388

### CODE REVIEW

File: .github/workflows/assistant-to-the-branch-manager.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: DevInfra
```
Comment (alan-agius4) [inline .github/workflows/assistant-to-the-branch-manager.yml:1]: This was added to avoid future conflicts.

Comment (devversion) [reply .github/workflows/assistant-to-the-branch-manager.yml:1]: thank you! Cc. @josephperrott

**Review by devversion — APPROVED** [review]

**Review by devversion — APPROVED** [review]
LGTM

Reviewed-for: global-dev-infra-approvers

**Review by josephperrott — APPROVED** [review]
LGTM

**Pull request closed without merging**